src/Transfer/Resources/Database/Attributes/FloatAttribute.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace Utopia\Transfer\Resources\Database\Attributes;
+
+use Utopia\Transfer\Resources\Database\Attribute;
+use Utopia\Transfer\Resources\Database\Collection;
+
+class FloatAttribute extends Attribute
```

Comment:
```suggestion
class Float extends Attribute
```

No need of the Attribute suffix. Make sure to change it in the file name.